packages/wonder-blocks-modal/util/maybe-get-portal-mounted-modal-host-element.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+import {ModalLauncherPortalAttributeName} from "./constants.js";
+
+/**
+ * From a given element, finds its next ancestor that is a modal launcher portal
+ * element.
+ * @param {null | Element | Text} element The element whose ancestors are to be
+ * walked.
+ * @returns {Element | null} The nearest parent modal launcher portal or
+ * `null`.
+ */
+function maybeGetNextAncestorModalLauncherPortal(element: null | Element | Text) {
+    let candidateElement = element ? element.parentElement : null;
+    while (
+        candidateElement &&
+        !candidateElement.hasAttribute(ModalLauncherPortalAttributeName)
```

Comment:
I thought you mentioned that in some browsers that eventually we might get to a situation where `candidateElement === candidateElement.parentElement`.  Was that actually not the case?

Reply:
After reading the docs on `parentElement` and `parentNode`, it is likely that I was confusing `parentElement` with `parentNode`. Based on re-reading the specs and some BrowserStack testing, I'm confident that we don't need that check.